cmds/bmp/update_state_command.go

```diff
@@ -45,10 +47,81 @@ func (cmd updateStateCommand) Options() cmds.Options {
 
 func (cmd updateStateCommand) Validate() (bool, error) {
 	cmd.printer.Printf("Validating %s command: options: %#v", cmd.Name(), cmd.options)
+
+	if cmd.options.Server == "" {
+		return false, errors.New("cannot have empty server ID")
+	}
+
+	if cmd.options.State == "" {
+		return false, errors.New("cannot have empty state")
+	} else {
+		if !isValidState(cmd.options.State) {
+			return false, errors.New("the server state is incorrect!")
+		}
+	}
+
 	return true, nil
 }
 
 func (cmd updateStateCommand) Execute(args []string) (int, error) {
 	cmd.printer.Printf("Executing %s command: args: %#v, options: %#v", cmd.Name(), args, cmd.options)
-	return 0, nil
+
+	cmd.ui.PrintlnInfo("Be careful to update the state of a server, which might cause Bluemix broken!")
```

Comment:
You should print this as something like:

```
cmd.ui.PrintlnInfo("WARNING: Be careful updating the state of a server, as it might break your deployment!")
```